akka-docs/rst/scala/code/docs/dispatcher/DispatcherDocSpec.scala

```diff
@@ -78,12 +78,8 @@ object DispatcherDocSpec {
       executor = "thread-pool-executor"
       # Configuration for the thread pool
       thread-pool-executor {
-        # minimum number of threads to cap factor-based core number to
-        core-pool-size-min = 2
-        # No of core threads ... ceil(available processors * factor)
-        core-pool-size-factor = 2.0
-        # maximum number of threads to cap factor-based number to
-        core-pool-size-max = 10
+        // or in Akka 2.4.2+
+        fixed-pool-size = 16
```

Comment:
yes, we can place the fixed size example first, but for completeness we should also include the factor example.

We can write

... with a fixed thread pool size (e.g. for IO bound tasks):

... with a thread pool size based on number of cores (e.g. for CPU bound tasks):